README.md

```diff
@@ -1,6 +1,43 @@
-# immutad●t
+immutad●t
+=================
+
+A JavaScript library to deal with nested immutable structures.
+
+```js
+push({ nested: { prop: [1, 2] } }, 'nested.prop', 3, 4)
+// → { nested: { prop: [1, 2, 3, 4] } }
+
+pickBy({ nested: [{ a: 1, b: 2, c: 3, d: 4 }, { e: 6 }] }, 'nested.0', v => v < 3)
```

Comment:
I find this one a little bit difficult to understand...
It's still a good example, and the push is easy to understand so I think it's ok.